### PR TITLE
New actionableProposalTotalCount Store

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

A new store to get a total number of all actionable proposals.

# Changes

- `actionableProposalTotalCountStore`

# Tests

- Added.

# Todos

- [ ] Add entry to changelog (if necessary).
Not needed.

### CODE REVIEW

File: frontend/src/lib/derived/actionable-proposals.derived.ts [hunk 1]
```diff
@@ -40,6 +40,13 @@ export const actionableProposalCountStore: Readable<ActionableProposalCountData>
     })
   );
 
+/** A store that contains a total count of all actionable proposals (nns + snses) */
+export const actionableProposalTotalCountStore: Readable<number> = derived(
+  actionableProposalCountStore,
+  (map) =>
+    Object.values(map).reduce((acc: number, count) => acc + (count ?? 0), 0)
```
Comment (dskloetd) [inline frontend/src/lib/derived/actionable-proposals.derived.ts:47]: Do you need `count ?? 0`? Is it possible for `count` to be `undefined`?
Can you add a test that demonstrates this?

Comment (mstrasinskis) [reply frontend/src/lib/derived/actionable-proposals.derived.ts:47]: Here is the source of `undefined` - https://github.com/dfinity/nns-dapp/pull/4803/files#diff-c53a13607f5fdb0011bd356eb51a2286c6a172bc235550db4e4a3f6ee56026d9R38.

A comment was added

Comment (dskloetd) [reply frontend/src/lib/derived/actionable-proposals.derived.ts:47]: --> Can you add a test that demonstrates this?

Comment (dskloetd) [reply frontend/src/lib/derived/actionable-proposals.derived.ts:47]: Sorry, I didn't realize the comment was in test that already does this. Thanks!

**Review by dskloetd — APPROVED** [review]

**Pull request merged**